### ISSUE TITLE
This is not an issue but a comment/suggestion

### ISSUE THREAD

**Issue by hatonthecat** [pc2005cz/TheUltimate486Upgrade#1]
Hi, I didn't see an e-mail so I figured you preferred inquiries via this method. Coreboot on a 486 sounds like an interesting application for a project idea I have: solar powered laptops. The 486 has around 1.2 million transistors, a foundry like GlobalFoundries/TSMC has a low-leakage node: ULL & FD-SOI. If this coreboot could be ported to a 486 node on 22nm, you could potentially develop a chip that runs on around 1mW and can boot linux/Windows 95.

**Comment by CodeAsm** [comment]
This seems more like a chip manufacturing question then coreboot. If you cant find a licensed design that pays intel, your legally out of luck im afraid.

Or find some industrial grade supply of 486based chips and you may have luck. Question seems out of the scope of this repo.

**Comment by hatonthecat** [comment]
@CodeAsm I'm aware that my initial post may have sounded out of scope. After seeing the detailed work in this repository, I think that the user pc2005cz seems to have a deep understanding of the level of research that I was seeking, and inquired if there may be interest to work on a new 486 design with modern i/o (e.g MMC), if the opportunity of a new chip manufacture arose. That does limit this project to 32-bit and 4GB, but is far more than I think would be needed for the simplest systems, and is meant to be modular and scalable- e.g 8MB to 128MB would probably be the range of the SoC board. It's similar in scope to the reviving the Quark concept, which was never sold with more than 1MB of RAM.

Regarding an x86 license, I have searched for definitive answers on other sites like Reddit, and some seem to think that 386 or 486 architectures are expired patents and don't require a license, whereas others say they still do (some say it's the abstract instruction set that is patented, rather than the method of implementation or technique). In any case, not everyone in those threads are referring to 486s but 586s and x64, so it's a long process. 

I did find a couple industrial /gen purpose processors:
https://www.vortex86.com/products
https://www.rdc.com.tw/
More which might be out of production are here:
https://en.wikipedia.org/wiki/List_of_x86_manufacturers

There are also open source ones too:
https://opencores.org/projects/ao486

In sum, the goal/interest is, whether a modern Quark-like TDP can be manufactured using open source implementations with a different process and integration for the DRAM and MMC for a bootloader.

**Comment by pc2005cz** [comment]
> Hi, I didn't see an e-mail so I figured you preferred inquiries via this method. Coreboot on a 486 sounds like an interesting application for a project idea I have: solar powered laptops. The 486 has around 1.2 million transistors, a foundry like GlobalFoundries/TSMC has a low-leakage node: ULL & FD-SOI. If this coreboot could be ported to a 486 node on 22nm, you could potentially develop a chip that runs on around 1mW and can boot linux/Windows 95.

Hi,
Well I'm usually online at retroweb discord. 

The cost of making a modern 486 chip would be probably too high to any retro use-case, but it should be possible. You could also put ao486 into an FPGA and use voltage translators to make socket 3 compatible CPU.

For low-power designs the x86 architecture isn't very efficient. I mean there are 16bit MCUs which claims nanowatt scale standby consumption.

**Comment by pc2005cz** [comment]
> This seems more like a chip manufacturing question then coreboot. If you cant find a licensed design that pays intel, your legally out of luck im afraid.
> 
> Or find some industrial grade supply of 486based chips and you may have luck. Question seems out of the scope of this repo.

Yeah I agree. This is just a blogpost about experiment which spanned from me being annoyed about 90s BIOS limits :-D

**Comment by hatonthecat** [comment]
> Hi, Well I'm usually online at retroweb discord.
> 
> The cost of making a modern 486 chip would be probably too high to any retro use-case, but it should be possible. You could also put ao486 into an FPGA and use voltage translators to make socket 3 compatible CPU.
> 
> For low-power designs the x86 architecture isn't very efficient. I mean there are 16bit MCUs which claims nanowatt scale standby consumption.

I'm working on the cost, by finding investors. The retro use I had in mind was porting it to a mobile platform, like a new motherboard the size of of a postage stamp, rather than Socket 3.

Nanowatt range isn't necessary to reach solar power. Microwatt usage in idle or even a few milliwatts would be fine and could run on a small hybrid capacitor with 90mAh- long enough for several hours using a 5-10mW screen and LTE radio.
